app/subscriber/package.json

```diff
@@ -42,7 +42,7 @@
     "sheetjs": "file:packages/xlsx-0.20.1.tgz",
     "styled-components": "6.1.11",
     "stylis": "4.3.2",
-    "tno-core": "0.1.105"
+    "tno-core": "0.1.108"
```

Comment:
I know presently we're at `0.1.110`.  You need to rebase your PR and publish a new tno-core package version.  Make sure you publish with the latest code changes within the `dev` branch.